src/pages/Items/Items.js

```diff
@@ -0,0 +1,215 @@
+`;
+
+const Icon = styled.img`
+  position: absolute;
+  padding: 13px 20px;
+`;
+
+const StyledButton = styled(Button)`
+  width: 133px;
+`;
+
+const AllProducts = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: repeat(2, auto);
+  gap: 24px;
+`;
+
+const PageNationIconContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`;
+
+function Items() {
+  const [products, setProducts] = useState([]);
+  const [page, setPage] = useState(1);
+  const [orderBy, setOrderBy] = useState("recent");
+
+  const handleRecent = () => {
+    setOrderBy("recent");
+  };
+
+  const hanldeFavorite = () => {
+    setOrderBy("favorite");
+  };
+
+  const hanldeLoad = async (options) => {
+    const { list } = await getItems(options);
+    if (options.page === 1) {
+      setProducts(list);
+    } else {
+      setProducts([...products, ...list]);
+    }
+    setPage(options.page + 1);
+  };
+
+  const handleLoadMore = async () => {
+    setPage((prev) => prev + 1);
+    await hanldeLoad({ page, pageSize: PAGESIZE, orderBy });
+  };
+
+  // orderBy 변경 시 페이지 초기화
+  useEffect(() => {
+    const initialPage = 1;
+    setPage(initialPage);
+    hanldeLoad({ page: initialPage, pageSize: PAGESIZE, orderBy });
+  }, [orderBy]);
+
+  return (
+    <StyledItems>
+      <BestProductsContainer>
+        <Title>베스트 상품</Title>
+        <BestProducts>
+          <ProductContainer
+            src={firstImg}
+            alt="아이패드 이미지"
+            description="아이패드 미니 팝니다"
+            price="500,000"
+            favoriteCount="1000"
+            width="282px"
+          />
+          <ProductContainer
+            src={secondImg}
+            alt="책 이미지"
+            description="책 팝니다"
+            price="50,000"
+            favoriteCount="800"
+            width="282px"
+          />
+          <ProductContainer
+            src={thirdImg}
+            alt="세탁기 이미지"
+            description="세탁기 팝니다"
+            price="500,000"
+            favoriteCount="700"
+            width="282px"
+          />
+          <ProductContainer
+            src={fourthImg}
+            alt="오븐 이미지"
+            description="오븐 팝니다"
+            price="300,000"
+            favoriteCount="500"
+            width="282px"
+          />
+        </BestProducts>
+      </BestProductsContainer>
+
+      <AllProductsContainer>
+        <AllProductsHeader>
+          <Title>전체 상품</Title>
+          <InputContainer>
+            <InputForm>
+              <Input placeholder="검색할 상품을 입력해주세요" />
+              <Icon src={readingGlasses} />
+              <Link to="/additem">
+                <StyledButton>상품 등록하기</StyledButton>
+              </Link>
+              <Accordian
+                handleRecent={handleRecent}
+                hanldeFavorite={hanldeFavorite}
+              />
+            </InputForm>
+          </InputContainer>
+        </AllProductsHeader>
+
+        <AllProducts>
+          {products.map((product) => {
+            return (
```

Comment:
```suggestion
          {products.map((product) => (
```
이런식으로 single body라면 brace빼주고 return 키워드도 없애줄수있답니다! 
좀더 간결하죠?